src/textEditor.ts

```diff
@@ -140,6 +141,32 @@ export class TextEditor {
     return vscode.window.activeTextEditor.document.getText(selection);
   }
 
+  /**
+   *  Retrieves the current word at position.
+   *  If current position is whitespace, selects the right-closest word
+   */
+  static getWord(position: Position) : string {
+    const whitespaceRegex = new RegExp("^ *$");
```

Comment:
Now that we have a settings object, could we put this in there? I don't think anyone sane will ever want to change it, but it'll be nice to have in one location. (I have a bunch of TODO(whitespace) in anticipation of this moment, if you wanna clean those up as well.)